src/locales/de-DE.json

```
@@ -78,14 +78,19 @@
       "addedSuccessfully": "{firstname} {lastname} wurde erfolgreich hinzugefügt.",
       "addNew": "Neuen Benutzer hinzufügen",
       "changePassword": "Passwort ändern",
-      "twoFactorAuthentication":{
+      "twoFactorAuthentication": {
         "setUpShort": "2FA einrichten",
         "setUpLong": "Zwei-Faktor-Authentifizierung einrichten",
         "header": "Zwei-Faktor-Authentifizierung (2FA)",
-        "softwareTokenOption":"Software-Token einrichten",
-        "softwareTokenText": "(i) Ein QR-Code wird generiert, welcher direkt eingescannt oder ausgedruckt werden kann.",
-        "hardwareTokenOption":"Hardware-Token zuordnen",
-        "hardwareTokenText": "(i) Der Hardware-Token muss vorliegen und funktionieren. Bitte notieren Sie, wem Sie den Token gegeben haben."
+        "softwareTokenOption": "Software-Token einrichten",
+        "softwareTokenText": "Ein QR-Code wird generiert, welcher direkt eingescannt oder ausgedruckt werden kann.",
+        "hardwareTokenOption": "Hardware-Token zuordnen",
+        "hardwareTokenText": "Der Hardware-Token muss vorliegen und funktionieren. Bitte notieren Sie, wem Sie den Token gegeben haben.",
+        "scanOrPrint": "QR-Code scannen oder ausdrucken.",
+        "notSetUp": "Für diesen Benutzer ist aktuell keine 2FA eingerichtet.",
+        "softwareTokenIsSetUp": "Für diesen Benutzer ist aktuell ein Software-Token eingerichtet.",
+        "resetInfo": "Um einen neuen Token einzurichten, muss der aktuelle Token durch die schulischen Administratorinnen und Administratoren zurückgesetzt werden.",
```

```suggestion
        "resetInfo": "Um einen neuen Token einzurichten, muss der aktuelle Token durch die Schuladmins zurückgesetzt werden.",
```
Ich bin mir relativ sicher, dass wir hier abkürzen dürfen. Mir wäre es lieb, wenn Christian die Texte einmal kurz abnimmt, da er im stetigen Austausch zum Kunden steht.

---

Hab nochmal nachgefragt und ja wir dürfen Abkürzen :)